crates/ruff_python_parser/tests/snapshots/valid_syntax@ambiguous_lpar_with_items_if_expr.py.snap

```diff
@@ -1,7 +1,6 @@
 ---
 source: crates/ruff_python_parser/tests/fixtures.rs
 input_file: crates/ruff_python_parser/resources/inline/ok/ambiguous_lpar_with_items_if_expr.py
-snapshot_kind: text
```

Comment:
Do we know where these edits to the snapshot files are coming from? They don't look relevant to the changes you've made to the generation script

Reply:
This is happening depending on what version of cargo-insta you have locally when running the tests(more context https://github.com/mitsuhiko/insta/issues/676). I'll go ahead and revert back to keep it tidy.